admin/controllers/administrator.js

```diff
@@ -0,0 +1,104 @@
+const Settings = require('../models/setting')
+const SettingsLog = require('../models/setting-log')
+const settingsErrorMessages = require('../lib/errors/settings')
+const settingsValidator = require('../lib/validator/settings-validator')
+const config = require('../config')
+
+const getAdministration = async (req, res, next) => {
```

Comment:
I'll add it, what should we do about the rest of the files in the project? None of them has jsDoc